### ISSUE TITLE
📝 Documentation: Add konamimojisplosion to the docs site

### ISSUE THREAD

**Issue by JoshuaKGoldberg** [JoshuaKGoldberg/ts-api-utils#142]
### Bug Report Checklist

- [X] I have pulled the latest `main` branch of the repository.
- [X] I have [searched for related issues](https://github.com/JoshuaKGoldberg/ts-api-utils/issues?q=is%3Aissue) and found none that matched my issue.

### Overview

Following up on #70: [Konamimojisplosion](https://github.com/JoshuaKGoldberg/Konamimojisplosion) is a fun project of mine that hooks up [konami-code-js](https://github.com/Haeresis/konami-code-js) to trigger [emojisplosion](https://github.com/JoshuaKGoldberg/emojisplosion). Let's add it to the docs site!

### Additional Info

In other words, I want to make it so that if someone types the [Konami Code](https://en.wikipedia.org/wiki/Konami_Code) on the https://joshuakgoldberg.github.io/ts-api-utils site, fireworks blasts of emojis will start popping up everywhere. 🎉 🎊

**Comment by JoshuaKGoldberg** [comment]
I just published [`typedoc-plugin-konamimojisplosion`](https://github.com/JoshuaKGoldberg/typedoc-plugin-konamimojisplosion). So I'll just take this on now.
